packages/web-components/src/components/content-block/content-block.scss

```diff
@@ -21,6 +22,10 @@
   display: block;
 }
 
+:host(#{$dds-prefix}-content-block) ::slotted([slot='footer']) {
+  background-color: pink;
```

Comment:
is this background color intentional?

Reply:
oh crap, was testing if that selector worked! thanks for catching that!